build-scripts/for-win64/prepare-debian-based.sh

```diff
@@ -17,11 +17,26 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
 sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
   wget unzip cmake g++ pkg-config g++-mingw-w64-x86-64 nsis \
   docbook-xsl xsltproc gettext po4a imagemagick zip libz-mingw-w64-dev \
-  wine32 winbind
+  wine32 winbind pipx
```

Comment:
```suggestion
  wine32 winbind pipx git
```

otherwise ``pipx install git+`` doesn not work

Reply:
pipx is installed in [the latest commit](https://github.com/GrandOrgue/grandorgue/blob/d0ad3014f1da880b057cbb129a682ca266c76746/build-scripts/for-win64/prepare-debian-based.sh).

Reply:
But it cannot install anything from git+...  unless  git is also installed.